src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/NetworkTopology.java

```diff
@@ -213,7 +229,8 @@ private void speakerRulesRouter(TopologyBuilder topology) {
 
     private void networkHistory(TopologyBuilder topology) {
         NetworkHistory spout = new NetworkHistory(persistenceManager);
-        declareSpout(topology, spout, NetworkHistory.SPOUT_ID);
+        declareBolt(topology, spout, NetworkHistory.BOLT_ID)
+                .allGrouping(ComponentId.INPUT_ZOOKEEPER.toString());
```

Comment:
You have declared Zookeeper spout as `ZooKeeperSpout.BOLT_ID` but all stream refer it as `ComponentId.INPUT_ZOOKEEPER.toString()`. Looks incorrect.

Reply:
Fixed